internal/npm_install/npm_install.bzl

```diff
@@ -226,6 +226,9 @@ npm_install = repository_rule(
     implementation = _npm_install_impl,
 )
 """Runs npm install during workspace setup."""
+# Adding the above docstring as `doc` attribute causes a build
```

Comment:
I raised this with @c-parsons

stardoc also doesn't generate any docs for repository rules which is why I haven't switched us over from skydoc

Reply:
Stardoc does generate docs for repository rules as of the current release. repository_rule also has a "doc" parameter as of the most recent Bazel release.

Reply:
Great. This repo just got updated to 0.24.1 so we can update to stardoc which will also fix this issue: https://github.com/bazelbuild/buildtools/issues/471#issuecomment-485283200

Reply:
Will leave that for another PR